src/datatypes/array.c

```diff
@@ -50,6 +50,18 @@ bool SIArray_ContainsType(SIValue siarray, SIType t) {
 	return false;
 }
 
+bool SIArray_ContainsValue(SIValue siarray, SIValue value) {
+	uint array_len = SIArray_Length(siarray);
+	for(uint i = 0; i < array_len; i++) {
+		SIValue elem = SIArray_Get(siarray, i);
```

Comment:
Although this is a good practice to use the API, I think that while in the same C file we can access the datatype internals (which is a bit cheaper) and work with the underline array
```c
SIValue elem = siarray.array[i];
```

Reply:
Ok. Change done in [aa6194f](https://github.com/RedisGraph/RedisGraph/pull/2807/commits/aa6194fbec956a35195dd72b6149b5636e2208a7)